test/unit/ngsiv2/contextRequests/singleMeasuresRawTypes.json

```diff
@@ -1,12 +1,6 @@
 {
   "humidity":{
     "type": "degrees",
-     "value": { 
-         "type": "Buffer",
-         "data": [
-             51,
-             50
-         ]
-     }
+     "value": "3332"
   }
 }
```

Comment:
Analysing this again, the test that is expecting this answer is:
https://github.com/telefonicaid/iotagent-json/blob/36bcc64fee817c4289f048b9ded3b2ea6fe5f4d9/test/unit/ngsiv2/HTTP_receive_measures-test3.js#L190-L205

Character 3 and 2 are `0x33` and `0x32` in hex representation.

Since the expected result is:
```
     "value": "3332"
````

It seems is the expected result.